Week1/homework/world-queries.txt

```diff
@@ -1,40 +1,362 @@
+| Virgin Islands, U.S.             |
++----------------------------------+
+37 rows in set (0.00 sec)
+
+---------------------------------------------------------------------------------
+4- What's the top 10 countries by SurfaceArea ?
+SELECT country.name, country.SurfaceArea FROM country ORDER BY SurfaceArea DESC LIMIT 10;
++--------------------+-------------+
+| name               | SurfaceArea |
++--------------------+-------------+
+| Russian Federation | 17075400.00 |
+| Antarctica         | 13120000.00 |
+| Canada             |  9970610.00 |
+| China              |  9572900.00 |
+| United States      |  9363520.00 |
+| Brazil             |  8547403.00 |
+| Australia          |  7741220.00 |
+| India              |  3287263.00 |
+| Argentina          |  2780400.00 |
+| Kazakstan          |  2724900.00 |
++--------------------+-------------+
+10 rows in set (0.00 sec)
+
+---------------------------------------------------------------------------------
+5- What's the top 10 most populated cities?
+ SELECT city.name, city.population FROM city ORDER BY population DESC LIMIT 10;
++-----------------------+------------+
+| name                  | population |
++-----------------------+------------+
+| Mumbai (Bombay)       |   10500000 |
+| Seoul                 |    9981619 |
+| Sï¿½o Paulo           |    9968485 |
+| Shanghai              |    9696300 |
+| Jakarta               |    9604900 |
+| Karachi               |    9269265 |
+| Istanbul              |    8787958 |
+| Ciudad de Mï¿½xico    |    8591309 |
+| Moscow                |    8389200 |
+| New York              |    8008278 |
++-----------------------+------------+
+10 rows in set (0.00 sec)
+
+---------------------------------------------------------------------------------
+6- Select the top 3 countries by population that their names start with the letter ‘P’?
+ SELECT country.name, country.population FROM country WHERE country.name LIKE 'p%' LIMIT 3;
++----------+------------+
+| name     | population |
++----------+------------+
+| Pakistan |  156483000 |
+| Panama   |    2856000 |
+| Pitcairn |         50 |
++----------+------------+
+3 rows in set (0.00 sec)
+
+---------------------------------------------------------------------------------
+7- What is the population in Asia?
+ SELECT Continent, SUM(population) AS 'TOTAL-POPULATION' FROM country WHERE country.continent = 'Asia';
++-----------+------------------+
+| Continent | TOTAL-POPULATION |
++-----------+------------------+
+| Asia      |       3705025700 |
++-----------+------------------+
+1 row in set (0.00 sec)
+
+---------------------------------------------------------------------------------
+8- Which languages are spoken in the Region of ‘South America’?
+SELECT 
+  DISTINCT 
+	countrylanguage.language,
+	countrylanguage.IsOfficial, 
+	country.region 
+  FROM 
+	countrylanguage, 
+	country 
+  WHERE  
+	country.region = 'South America'
+  AND
+	countrylanguage.countrycode = country.code 
```

Comment:
It's nice to see that you found a different way of doing joins. It is correct, but these days using an INNER JOIN and ON clause is more common, because it is considered more readable. If you have many conditions in the WHERE clause, the syntax you are using can become confusing.
This is an alternative answer to the question:

`SELECT distinct language 
FROM country c
JOIN countryLanguage cc ON c.code = cc.countryCode
WHERE region = 'South America';`